### PR TITLE
Menu and Footer changes

### PR DESCRIPTION
**Pull request by Vamsi027**
The commit has changes done in Menu bar and footer bar of the new home page design related to the issue. 
https://github.com/MU-DBL/fatplants/issues/23

### CODE REVIEW

File: fatplant/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -66,7 +67,8 @@ const routes: Routes = [{path: '', redirectTo: '/homepage', pathMatch: 'full'},
 {path: 'pathway-viewer', component: PathwayViewerComponent},
 {path: 'extended-pathway', component: ExtendedPathwayComponent},
 {path: 'blast-internal', component:BlastInternalComponent},
-{path: '**', redirectTo: '/homepage'}];
+{path: 'home', component:HomeComponent},
+{path: '**', redirectTo: '/home'}];
```
Comment (Yongfang-Qin) [inline fatplant/src/app/app-routing.module.ts]: we will continue to use the old one until we finish all the web pages. So please keep this `{path: '**', redirectTo: '/homepage'}];`

File: fatplant/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -32,8 +32,9 @@ import { ProteinSoybeanComponent } from './components/pages/datapages/protein-so
 import { ProteinCamelinaComponent } from './components/pages/datapages/protein-camelina/protein-camelina.component';
 import { ExtendedPathwayComponent } from './components/pages/tools/extended-pathway/extended-pathway.component';
 import { BlastInternalComponent } from './components/pages/tools/blast-internal/blast-internal.component';
+import { HomeComponent } from './components/homecomponents/home/home.component';
 
-const routes: Routes = [{path: '', redirectTo: '/homepage', pathMatch: 'full'},
+const routes: Routes = [{path: '', redirectTo: '/home', pathMatch: 'full'},
```
Comment (Yongfang-Qin) [inline fatplant/src/app/app-routing.module.ts]: will continue to use the old one until we finish all the web pages. So please keep this` const routes: Routes = [{path: '', redirectTo: '/homepage', pathMatch: 'full'}`

File: fatplant/src/app/app-routing.module.ts (hunk 1 above)
Comment (Yongfang-Qin) [reply fatplant/src/app/app-routing.module.ts]: My suggestion is that you create a new path for your `HomeComponent` and don't make changes to the old `homepage` routing

File: fatplant/src/app/app.component.css [hunk 3]
```diff
@@ -27,3 +27,8 @@ button:focus {
     overflow-x: hidden;
 }
 
+.scrollable-container {
+    height: calc(120vh - 100px);
```
Comment (Yongfang-Qin) [inline fatplant/src/app/app.component.css:31]: Good job for using responsive design

File: fatplant/src/app/app.component.html [hunk 4]
```diff
@@ -1,14 +1,14 @@
-<mat-sidenav-container autosize #mobileMenu id="sideNavContainer" class="absolute background">
```
Comment (Yongfang-Qin) [inline fatplant/src/app/app.component.html]: Like I said we don't make changes to old files

File: fatplant/src/app/app.component.ts [hunk 5]
```diff
@@ -1,36 +1,36 @@
-import { Component, ElementRef, ViewChild } from '@angular/core';
-import { MobileService } from './services/mobile/mobile.service';
-import { AuthService } from './services/auth.service';
-import { HeaderComponent } from './header/header.component';
```
Comment (Yongfang-Qin) [inline fatplant/src/app/app.component.ts]: is there a reason why you changed the single quotation to double quotations?

File: fatplant/src/index.html [hunk 6]
```diff
@@ -13,7 +13,8 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 </head>
-<body class="background">
+<!-- <body class="background"> -->
```
Comment (Yongfang-Qin) [inline fatplant/src/index.html]: Like I said, I will continue to use the old home page

File: fatplant/src/app/app.component.html [hunk 7]
```diff
@@ -45,4 +45,9 @@
 <mat-menu #graphMenu>
   <a mat-menu-item routerLink="/protein-network" routerLinkActive="active-nav" class="menu-item" (click)="sideNav.close()">Protein-Protein Network</a>
   <a mat-menu-item routerLink="/go-network" routerLinkActive="active-nav" class="menu-item" (click)="sideNav.close()">GO Enrichment Network</a>
-</mat-menu>
+</mat-menu> -->
+<div class="scrollable-container">
+  <app-menu></app-menu>
+  <router-outlet></router-outlet>
+  <app-footer></app-footer>
```
Comment (Yongfang-Qin) [inline fatplant/src/app/app.component.html:52]: do we need a footer for all web pages? If not, then put it on the home page only.

**Review by Vamsi027 — COMMENTED** [review]
I have done changes according to the review comments by restoring the old code. And as discussed we will put the footer for all pages. And there was a vs code feature which has changed few lines of code to double quotes. Now, it has been reverted back.

**Comment by Vamsi027** [comment]
New pull request created

**Pull request closed without merging**